### PR TITLE
Feature - Update createSite Page to use ui-react

### PR DESCRIPTION
**Pull request by MoiyadKayda**
## Screenshots:
image

## Checklist:

- [ ] I have updated the CHANGELOG with a summary of my changes

### CODE REVIEW

File: src/packages/console/src/pages/createSitePage.tsx [hunk 1]
```diff
@@ -143,99 +108,95 @@ export const CreateSitePage = (): React.ReactElement => {
...
+                      label='Site name'
+                      name='name'
+                      value={name}
+                      onValueChanged={onNameChanged}
+                      inputType={InputType.Text}
+                      inputWrapperVariant={nameError ? 'error' : ''}
+                      messageText={nameError}
+                      placeholderText='Site name'
+                    />
+                    <SingleLineInput
```
Comment (krishan711) [inline src/packages/console/src/pages/createSitePage.tsx]: this one used to actually be a select its just wierdly done via a textfield

Comment (MoiyadKayda) [reply src/packages/console/src/pages/createSitePage.tsx]: Instead of a input, we can have a button

Comment (MoiyadKayda) [reply src/packages/console/src/pages/createSitePage.tsx]: or a custom styled component

Comment (krishan711) [reply src/packages/console/src/pages/createSitePage.tsx]: well if you can make a custom styled component dropdown that would be the best cos we can turn it into a ui-react component later. but button wouldnt work cos you have to pick from a list of options

File: src/packages/console/src/pages/createSitePage.tsx [hunk 2]
```diff
@@ -143,99 +108,95 @@ export const CreateSitePage = (): React.ReactElement => {
...
+        <NavigationBar />
+        <Stack direction={Direction.Vertical} paddingVertical={PaddingSize.Wide2} isFullHeight={true}>
+          <Stack.Item growthFactor={1} shrinkFactor={1} />
+          <Box variant='card'>
+            <Stack direction={Direction.Vertical} shouldAddGutters={true}>
+              <Stack.Item alignment={Alignment.Center} gutterAfter={PaddingSize.Wide2}>
+                <Text tag='h1' variant='header5'>Create a new site</Text>
+              </Stack.Item>
+              {accounts === null ? (
+                <Text tag='p'>
```
Comment (krishan711) [inline src/packages/console/src/pages/createSitePage.tsx]: why p? also can you make it `variant={note}` so its a bit subtle. Also you can inline the text no need for the `{'...'}` on the next line

File: src/packages/console/src/pages/createSitePage.tsx (hunk 1 above)
Comment (MoiyadKayda) [reply src/packages/console/src/pages/createSitePage.tsx]: Actually I think a button or a hybrid will be good. As it opens up a model

Comment (MoiyadKayda) [reply src/packages/console/src/pages/createSitePage.tsx]: like a text+button
the currently selected templete is set as text
and when button is clicked it opens the model

Comment (krishan711) [reply src/packages/console/src/pages/createSitePage.tsx]: oh yes it opens a modal i forgot. yes try that.

Comment (MoiyadKayda) [reply src/packages/console/src/pages/createSitePage.tsx]: Ok

Comment (MoiyadKayda) [reply src/packages/console/src/pages/createSitePage.tsx:173]: Should I make a styled component

Comment (MoiyadKayda) [reply src/packages/console/src/pages/createSitePage.tsx:173]: or try to use the existing components in ui-react

Comment (MoiyadKayda) [reply src/packages/console/src/pages/createSitePage.tsx]: what should I use to create a flex div using ui-react

Comment (krishan711) [reply src/packages/console/src/pages/createSitePage.tsx]: depends what you want to do with it but generally stack

File: src/packages/console/src/pages/createSitePage.tsx [hunk 3]
```diff
@@ -131,9 +109,9 @@ export const CreateSitePage = (): React.ReactElement => {
     setIsAccountUpgradePopupShowing(false);
     history.navigate(`/accounts/${selectedAccountId}#plan`);
   };
-
-  const onTemplateChoiceClicked = (event: React.SyntheticEvent<HTMLDivElement>): void => {
-    event.preventDefault();
+  // event: React.SyntheticEvent<HTMLDivElement>
```
Comment (krishan711) [inline src/packages/console/src/pages/createSitePage.tsx]: remove comments pls

File: src/packages/console/src/pages/createSitePage.tsx [hunk 4]
```diff
@@ -143,99 +121,94 @@ export const CreateSitePage = (): React.ReactElement => {
...
+                      label='Site name'
+                      name='name'
+                      value={name}
+                      onValueChanged={onNameChanged}
+                      inputType={InputType.Text}
+                      inputWrapperVariant={nameError ? 'error' : ''}
+                      messageText={nameError}
+                      placeholderText='Site name'
+                    />
+                    <StyledBox bordered>
```
Comment (krishan711) [inline src/packages/console/src/pages/createSitePage.tsx:179]: does this not work if you just use `Box variant='bordered'>`?

Comment (MoiyadKayda) [reply src/packages/console/src/pages/createSitePage.tsx:179]: I tried using box but I was not able to reduce the padding in the box and also I was not able to give a left gutter when I was using box.

Comment (krishan711) [reply src/packages/console/src/pages/createSitePage.tsx:179]: the gutter can go on the stack and you can use `unpadded` i think? not totally sure but worth trying i think?

Comment (MoiyadKayda) [reply src/packages/console/src/pages/createSitePage.tsx:179]: I tried that, didn't work the way I wanted it to

Comment (krishan711) [reply src/packages/console/src/pages/createSitePage.tsx:179]: ok cool

Comment (MoiyadKayda) [reply src/packages/console/src/pages/createSitePage.tsx:179]: I have removed the comments

Comment (MoiyadKayda) [reply src/packages/console/src/pages/createSitePage.tsx:179]: Now just the select is using material-ui
So, if want I can try to make a dropdown select for ui-react

**Review by krishan711 — APPROVED** [review]

**Pull request merged**